src/extensions/extensionInterface.ts

```diff
@@ -233,7 +250,16 @@ type ScaleAmpScaleFactorRecordingSelectionAction = {
     multiplier: number
 }
 
-export type RecordingSelectionAction = SetRecordingSelectionRecordingSelectionAction | SetSelectedElectrodeIdsRecordingSelectionAction | SetCurrentTimepointRecordingSelectionAction | SetTimeRangeRecordingSelectionAction | SetAmpScaleFactorRecordingSelectionAction | ScaleAmpScaleFactorRecordingSelectionAction
+type SetCurrentTimepointVelocityRecordingSelectionAction = {
+    type: 'SetCurrentTimepointVelocity',
+    velocity: number // timepoints per second
+}
+
+type AnimateRecordingSelectionAction = {
+    type: 'AnimateRecording'
+}
+
+export type RecordingSelectionAction = SetRecordingSelectionRecordingSelectionAction | SetSelectedElectrodeIdsRecordingSelectionAction | SetCurrentTimepointRecordingSelectionAction | SetTimeRangeRecordingSelectionAction | SetAmpScaleFactorRecordingSelectionAction | ScaleAmpScaleFactorRecordingSelectionAction | SetCurrentTimepointVelocityRecordingSelectionAction | AnimateRecordingSelectionAction
```

Comment:
Clearly a lot of the recent changes over the last couple weeks have had to do with putting together the `RecordingSelection` object and associated Reducer; this seems like a good potential jumping-off point, or at least something worth featuring prominently in new documentation.